trie-db/src/lib.rs

```diff
@@ -384,44 +403,58 @@ pub trait TrieLayout {
 	const USE_EXTENSION: bool;
 	/// If true, the trie will allow empty values into `TrieDBMut`
 	const ALLOW_EMPTY: bool = false;
+	/// Threshold above which an external node should be
+	/// use to store a node value.
+	const MAX_INLINE_VALUE: Option<u32>;
+
 	/// Hasher to use for this trie.
 	type Hash: Hasher;
 	/// Codec to use (needs to match hasher and nibble ops).
 	type Codec: NodeCodec<HashOut=<Self::Hash as Hasher>::Out>;
 }
 
+/// Small enum indicating representation of a given children.
+pub enum ChildrenDecoded {
+	/// Children is stored as a node.
+	Hash,
+	/// Children is stored as an inline node.
+	Inline,
+	/// No children is present.
+	None,
+}
+
```

Comment:
Where is this struct used ?

```suggestion
```

Reply:
:+1: Probably some uncleaned code, removed it.